frontend/templates/product/sections/CrossSellSection/index.tsx

```diff
@@ -303,137 +313,155 @@ function CrossSellItem({
+                        w="full"
+                        direction={{
+                           base: 'column',
+                        }}
+                        ml={{
+                           base: 3,
+                           md: 0,
+                        }}
+                     >
+                        {isCurrentItem && (
+                           <HStack
+                              position={{
+                                 base: 'relative',
+                                 md: 'absolute',
+                              }}
+                              top={{
+                                 base: 'auto',
+                                 md: 4,
+                              }}
+                              right={{
+                                 base: 'auto',
+                                 md: 4,
+                              }}
+                              spacing="1"
+                              mb="3"
+                           >
+                              <Badge colorScheme="brand">Current item</Badge>
+                           </HStack>
+                        )}
+                        <Flex direction="column" h="full" align="flex-start">
+                           <Text
+                              fontSize="md"
+                              mb="2"
+                              _groupHover={{ color: 'brand.500' }}
+                           >
+                              {product.title}
+                           </Text>
+                           {product.rating != null &&
+                              product.reviewsCount != null && (
+                                 <ProductRating
+                                    mb="3"
+                                    rating={product.rating}
+                                    count={product.reviewsCount}
+                                 />
+                              )}
+                        </Flex>
+                     </Flex>
+                     <Box
+                        position={{
+                           base: 'relative',
+                           md: 'absolute',
+                        }}
+                        pl={{
+                           base: 3,
+                           md: 0,
+                        }}
+                        top={{
+                           base: 0,
+                           md: 4,
+                        }}
+                     >
+                        <Center
+                           onClick={(event) => {
+                              onChange(!isSelected);
+                              event.preventDefault();
+                              event.stopPropagation();
                            }}
-                           top={{
-                              base: 'auto',
-                              md: 4,
+                           onKeyDown={(event) => {
+                              if (['Enter'].includes(event.code)) {
```

Comment:
Is there a reason for this odd condition? It seems an odd way to do an equality test.

Reply:
Good eye I didn't notice that. Looking at other examples of this pattern in this repo, it seems to be used when we expect to add another value to the array in the future. Given that we probably wouldn't add any other keypress to this array, I agree that checking equality with `===` makes more sense.